examples/fibonacci/guest/src/lib.rs

```diff
@@ -1,15 +1,19 @@
 #![cfg_attr(feature = "guest", no_std)]
+use jolt::{end_cycle_tracking, start_cycle_tracking};
 
 #[jolt::provable]
 fn fib(n: u32) -> u128 {
     let mut a: u128 = 0;
     let mut b: u128 = 1;
     let mut sum: u128;
+
+    start_cycle_tracking("fib_loop"); // Use `start_cycle_tracking("{name}")` to start a cycle span
```

Comment:
Would be nice to have a proc macro that would attribute a function or a block, 
```rust
#[cycle_track(label="optional_label")]
fn calculation() {...}
```
@moodlezoup what do you think?

Reply:
good idea, it would be nice to closely match the `tracing` API. Could you write up a Linear issue for it?